StencylWorks.prefs

```diff
@@ -0,0 +1,4 @@
+global.engine.extension.style=true
+global.extension.style=true
+global.workdir.style=true
+global.workdir=/~
```

Comment:
@erikos Seems like just `/` works as well. It does not resolve `~/` as the home directory, but `/` seems to do the trick as well.